app/components/post_priority/post_priority_label.tsx

```diff
@@ -2,68 +2,42 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
-import {useIntl} from 'react-intl';
-import {type StyleProp, StyleSheet, Text, View, type ViewStyle} from 'react-native';
+import {defineMessages} from 'react-intl';
 
-import CompassIcon from '@components/compass_icon';
-import {PostPriorityColors, PostPriorityType} from '@constants/post';
-import {typography} from '@utils/typography';
+import Tag from '@components/tag';
+import {PostPriorityType} from '@constants/post';
 
-const style = StyleSheet.create({
-    container: {
-        alignSelf: 'flex-start',
-        flexDirection: 'row',
-        borderRadius: 4,
-        alignItems: 'center',
-        paddingHorizontal: 4,
-    },
+type Props = {
+    label: PostPriority['priority'];
+};
+
+const messages = defineMessages({
     urgent: {
-        backgroundColor: PostPriorityColors.URGENT,
+        id: 'post_priority.label.urgent',
+        defaultMessage: 'URGENT',
     },
     important: {
-        backgroundColor: PostPriorityColors.IMPORTANT,
-    },
-    label: {
-        color: '#fff',
-        ...typography('Body', 25, 'SemiBold'),
-    },
-    icon: {
-        color: '#fff',
-        fontSize: 12,
-        marginRight: 4,
+        id: 'post_priority.label.important',
+        defaultMessage: 'IMPORTANT',
     },
 });
 
-type Props = {
-    label: PostPriority['priority'];
-};
-
 const PostPriorityLabel = ({label}: Props) => {
-    const intl = useIntl();
-
-    const containerStyle: StyleProp<ViewStyle> = [style.container];
-    let iconName = '';
-    let labelText = '';
-    if (label === PostPriorityType.URGENT) {
-        containerStyle.push(style.urgent);
-        iconName = 'alert-outline';
-        labelText = intl.formatMessage({id: 'post_priority.label.urgent', defaultMessage: 'URGENT'});
-    } else if (label === PostPriorityType.IMPORTANT) {
-        containerStyle.push(style.important);
-        iconName = 'alert-circle-outline';
-        labelText = intl.formatMessage({id: 'post_priority.label.important', defaultMessage: 'IMPORTANT'});
+    if (label === PostPriorityType.STANDARD) {
+        return null;
     }
+
+    const isUrgent = label === PostPriorityType.URGENT; // else it is important
```

Comment:
the comment "else it is important" doesn't really help, IMO.

(0/5)

Reply:
There are three states: "Standard" (already filtered out at the beginning of the function), "Urgent" and "Important". I think it is better to keep things clear that the meaning of `!isUrgent` is that it is important.

But happy to rethink this.